rest/src/main/groovy/whelk/rest/api/SearchUtils2.java

```diff
@@ -154,7 +156,11 @@ public Map<String, Object> getPartialCollectionView(Map<String, Object> esRespon
         private SimpleQueryTree getFilteredTree() {
             var filters = new ArrayList<>(DEFAULT_FILTERS);
             if (object != null) {
-                filters.add(SimpleQueryTree.pvEqualsLiteral("_links", object));
+                if (predicates.isEmpty()) {
+                    filters.add(SimpleQueryTree.pvEqualsLink("_links", object));
```

Comment:
`object` should actually be wrapped as a Literal rather than Link here since `_links` is just a list of strings (no proper `@id` links)